scene/2d/canvas_item.cpp

```diff
@@ -356,6 +356,9 @@ bool CanvasItem::_edit_is_selected_on_click(const Point2 &p_point, double p_tole
 	if (_edit_use_rect()) {
 		return _edit_get_rect().has_point(p_point);
 	} else {
+		if (get_script_instance() && get_script_instance()->has_method("_edit_is_selected_on_click")) {
+			return get_script_instance()->call("_edit_is_selected_on_click", Vector2(p_point.x, p_point.y));
```

Comment:
```suggestion
			return get_script_instance()->call("_edit_is_selected_on_click", p_point);
```
`Point2` is an alias (`typedef`) of `Vector2`, so likely no need to convert here.